pipeline/tasks/gapic_tasks.py

```diff
@@ -105,15 +105,24 @@ def validate(self):
         return [gapic_requirements.GapicRequirements]
 
 
-class GapicCopyTask(task_base.TaskBase):
-    """Simply copy the files generated by Gapic toolkit to the final_repo_dir.
+class GapicCleanTask(task_base.TaskBase):
+    """Delete all of the files in the final_repo_dir.
     """
 
     def execute(self, final_repo_dir, gapic_code_dir):
-        # Simply removes the previous code generation results.
         self.exec_command(['rm', '-rf', final_repo_dir])
-        self.exec_command([
-            'cp', '-rf', gapic_code_dir, final_repo_dir])
+
+
+class GapicCopyTask(task_base.TaskBase):
+    """Copy the files generated by Gapic toolkit to the final_repo_dir.
+    """
+
+    def execute(self, final_repo_dir, gapic_code_dir):
+        print "Copying " + gapic_code_dir + "/* to " + final_repo_dir
+        for entry in os.listdir(gapic_code_dir):
```

Comment:
I already interfered with python in a previous PR when I renamed this output directory... Anyway, python's final output dir is under output/ , so it currently has to be manually copied.